ElrondNetwork/elrond-go
# Added trie sync speed in prints

- added trie sync speed & total data received

testing scenario: 
1. start a node connected to the testnet, on shard 1
2. observe the prints that should show like:
```
INFO [2021-11-04 10:20:48.137]   trie sync in progress                    name = user accounts for shard 1 num received = 727634 num large nodes = 0 num missing = 28688 num tries = 1389/1395 intercepted trie nodes cache size = 100.00 MB num of intercepted trie nodes = 2738452 data size received = 77.78 MB network speed = 971.20 KB/s
``` 
where there are 2 new info zones: `data size received` and `network speed`
3. when the trie sync finishes, the print should contain the peak speed attained like:
```
INFO [2021-11-04 10:28:05.869]   finished trie sync                       name = user accounts for shard 1 num received = 7864760 num large nodes = 0 num missing = 0 data size received = 942.45 MB peak network speed = 5.37 MB/s average network speed = 867.14 KB/s 
```
where there are 3 new info zones: `data size received`, `peak network speed` and `average network speed`
